zingoconfig/src/lib.rs

```diff
@@ -393,34 +392,7 @@ impl ZingoConfig {
         log_path.into_boxed_path()
     }
 
-    /// Coin Types are specified in public registries to disambiguate coin variants
```

Comment:
Let's this deprecate these first, and delete after some grace period.

Reply:
good idea i agree